install.sh

```diff
@@ -103,13 +103,15 @@ if [[ $(docker volume ls -q --filter name=dispatch-postgres) && $(docker run --r
     docker volume rm dispatch-postgres-new
 fi
 
-echo ""
-echo "Setting up database..."
-if [ $CI ]; then
-  docker-compose run --rm dispatch database upgrade --noinput
-else
-  docker-compose run --rm dispatch database upgrade
-fi
+# TODO Fix database upgrade/migration step
+# https://github.com/Netflix/dispatch-docker/issues/11
+#echo ""
+#echo "Setting up database..."
+#if [ $CI ]; then
+#  docker-compose run --rm web database upgrade --noinput
```

Comment:
If you used web, you receive this error - 
`database: 1: exec: /etc/dispatch/docker-entrypoint.sh: Permission denied`

If you replaced web with postgres, you receive this error - 
`/usr/local/bin/docker-entrypoint.sh: line 311: exec: database: not found`

If you used dispatch-sheduler, you receive this error - 
`database: 1: exec: /etc/dispatch/docker-entrypoint.sh: Permission denied`

Reply:
For problem 1 and 3, please ensure you are using a recent version of the `develop` branch in https://github.com/Netflix/dispatch

This has been addressed a handful of times:

- https://github.com/Netflix/dispatch/pull/51
- https://github.com/Netflix/dispatch/pull/53

> If you replaced web with postgres, you receive this error -

Yeah, that seems to make sense. `database upgrade` is actually a command passed to the `dispatch` binary, which shouldn't be available in the `postgres` container.

Reply:
Thank you, I was able to get a step further with `web`, but then get this - 
```
Setting up database...
Creating network "dispatch_default" with the default driver
Creating dispatch_postgres_1 ... done
Traceback (most recent call last):
  File "/usr/local/bin/dispatch", line 5, in <module>
    from dispatch.cli import entrypoint
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 12, in <module>
    from dispatch import __version__, config
  File "/usr/local/lib/python3.8/site-packages/dispatch/config.py", line 43, in <module>
    DISPATCH_HELP_EMAIL = config("DISPATCH_HELP_EMAIL")
  File "/usr/local/lib/python3.8/site-packages/starlette/config.py", line 59, in __call__
    return self.get(key, cast, default)
  File "/usr/local/lib/python3.8/site-packages/starlette/config.py", line 72, in get
    raise KeyError(f"Config '{key}' is missing, and has no default.")
KeyError: "Config 'DISPATCH_HELP_EMAIL' is missing, and has no default."```

Where should the .env exist with these configs? I set the configs within this file - https://github.com/Netflix/dispatch-docker/blob/master/.env but I still get this error.

Reply:
Copy `.env.example` to `.env`

They're all dummy values and the app still won't run, but the config errors will quiet down.

Replace them with real values if you have them and see what happens!

Reply:
I clone down this repository and copy the `.env.example` to `.env` within the same directory as `install.sh`. I also added values for DISPATCH_HELP_EMAIL and DISPATCH_HELP_SLACK_CHANNEL and I still get the error. I tried the same with putting the `.env` within the docker directory and still get the error.